style.css

```diff
@@ -0,0 +1,472 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+
+.hero-section {
+  background-image: linear-gradient(
+      to top,
+      rgba(0, 0, 0, 0.8) 0,
+      rgba(0, 0, 0, 0) 60%,
+      rgba(0, 0, 0, 0.8) 85%
+    ),
+    url(images/background.jpg);
+
+  background-size: cover;
+  position: relative;
+  background-position: center;
+  /* height: 700px; */
+}
+
+.container {
+  max-width: 1004px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 20px 0;
+}
+
+.headerLogo {
+  max-width: 148px;
+}
+
+.netflixLogo {
+  width: 100%;
+}
+
+.header-content {
+  display: flex;
+}
+
+.language-content {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  color: #fff;
+  border: 1px solid rgba(255, 255, 255, 0.459);
+  padding: 5px 10px;
+  border-radius: 3px;
+}
+
+.language-content select {
+  appearance: none;
+  background: transparent;
+  color: #fff;
+  border: none;
+  font-size: 16px;
+}
+
+.down-icon {
+  max-width: 10px;
+  width: 100%;
+}
+
+.signIn-btn {
+  background: #e50914;
+  color: white;
+  border: none;
+  padding: 5px 15px;
+  border-radius: 5px;
+  margin-left: 20px;
+}
+
+.signIn-btn:hover {
+  cursor: pointer;
+  background: rgb(153, 22, 29);
+}
+
+.signIn-btn a {
+  font-size: 14px;
+  font-weight: 600;
+  color: white;
+  text-decoration: none;
+}
+
+.main-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 150px 0;
+  color: white;
+}
+
+.main-heading {
+  text-align: center;
+  font-size: 43px;
```

Comment:
same as above when you use font size than should be cover with weight and line height both